portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments")
+public final class CommentServlet extends HttpServlet {
+
+    private List<Comment> comments;
+
+    @Override
+    public void init() {
+        comments = new ArrayList<>();
+
+        Comment comment1 = new Comment("Anonymous", "This is a comment");
```

Comment:
Do you have to initialise comments?

Reply:
I think I do. I can't add comments to the array list without first initialising an array list. I tried to erase that line, and it gave me a null pointer exception because I can't add comments to a list that does not exist.

Reply:
What about just having "comments = new ArrayList<>();"?

Reply:
Oh yeah, that's what I meant. The hardcoded comments are just there for this step.